learning_journal/forms.py

```diff
@@ -10,4 +10,15 @@ class EntryCreateForm(Form):
     body = TextAreaField(
         'Entry body',
         [validators.Length(min=1)],
+        filters=[strip_filter])
+# use inheritance here?
+class EntryEditForm(Form):
+    id = HiddenField('Entry ID') # add validators?
+    title = TextField(
+        'Entry title',
+        [validators.Length(min=1, max=255)],
+        filters=[strip_filter])
+    body = TextAreaField(
+        'Entry body',
+        [validators.Length(min=1)],
```

Comment:
Notice how your `EditEntryForm is identical to your `EntryCreateForm` in every respect, *except* the id field?  You've repeated a lot of lines of code here that you did not need to.  Can you think of a way to avoid that?